src/main/java/frc/robot/commands/AdjustScoreMode.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class AdjustScoreMode extends CommandBase {
+  public static int m_ScoreMode = 0;
+  /** Creates a new increaseScoreMode. */
+  public AdjustScoreMode() {
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (m_ScoreMode == 3) {
+      m_ScoreMode = 0;
+    } else m_ScoreMode++;
+  }
```

Comment:
I guess I am not quite following this command, would you mind walking me through how it should work?

Reply:
Basically, I created a separate command to increment the score mode variable so I could map it to a button. This would allow me to keep the score function condensed while still allowing easy access to the variable.

Reply:
This is as opposed to having different instances of a score command with different parameters as we talked about yesterday?